core/src/commonMain/kotlin/io/tolgee/Tolgee.kt

```diff
@@ -141,46 +147,46 @@ open class Tolgee(
     }
 
     /**
-     * Updating Tolgee translation for key with parameters.
+     * Updating Tolgee translation for a key with parameters.
      *
      * Respects locale changes from [setLocale].
      */
     @JvmOverloads
     @OptIn(ExperimentalCoroutinesApi::class)
     @NativeCoroutines
-    open fun translation(
+    open fun tFlow(
         key: String,
         parameters: TolgeeMessageParams = TolgeeMessageParams.None
     ): Flow<String> = localeFlow.mapLatest { locale ->
         val translation = currentTranslation(locale) ?: suspendCatching {
             loadTranslations(locale)
-        }.getOrNull() ?: currentTranslation(locale) ?: return@mapLatest instant(key, parameters)
+        }.getOrNull() ?: currentTranslation(locale) ?: return@mapLatest t(key, parameters)
 
         return@mapLatest translation.localized(key, parameters, locale)
-    }.mapNotNull()
+    }.filterNotNull()
 
     @OptIn(ExperimentalCoroutinesApi::class)
     @NativeCoroutines
-    open fun stringArrayTranslation(
+    open fun tArrayFlow(
         key: String
     ): Flow<List<String>> = localeFlow.mapLatest { locale ->
         val translation = currentTranslation(locale) ?: suspendCatching {
             loadTranslations(locale)
-        }.getOrNull() ?: currentTranslation(locale) ?: return@mapLatest stringArrayInstant(key)
+        }.getOrNull() ?: currentTranslation(locale) ?: return@mapLatest tArray(key)
 
         return@mapLatest translation.stringArray(key, locale)
-    }.mapNotNull()
+    }.filterNotNull()
 
     /**
-     * Immediate Tolgee translation for key with parameters.
+     * Immediate Tolgee translation for a key with parameters.
      *
-     * **Requires** calling [preload] or [translation] before.
+     * **Requires** calling [preload] or [tFlow] before.
      * (Otherwise) May return null if no translations are loaded at time calling.
      *
      * Respects only the locale at time calling.
      */
     @JvmOverloads
-    open fun instant(
+    open fun t(
```

Comment:
Are you sure you want to name these methods just `t`?

This is super unusual, I know it's pretty common in the web world, but for Java and Kotlin it's normally a no go.